mobile/src/modules/transactions/screens/SendScreen.tsx

```diff
@@ -55,12 +58,25 @@ class Send extends React.PureComponent<Props, State> {
       this.deepLinkProps = {
         sender: null,
         address: params.receiver,
-        fee: convertNQTStringToNumber(params.feeNQT).toString(),
-        amount: convertNQTStringToNumber(params.amountNQT).toString()
+        fee: this.getFee(params.feeNQT, params.feeSuggestionType),
+        amount: convertNQTStringToNumber(params.amountNQT).toString(),
+        message: params.message,
+        messageIsText: params.messageIsText === 'false' ? false : true,
```

Comment:
how dare you -> what wrong with: `params.messageIsText === 'true'`?

Reply:
because it needs to be true by default